packages/eslint-plugin/src/rules/no-confusing-void-expression.ts

```diff
@@ -167,6 +184,26 @@ export default createRule<Options, MessageId>({
         if (invalidAncestor.type === AST_NODE_TYPES.ReturnStatement) {
           // handle return statement
 
+          if (options.ignoreVoidInVoid) {
+            if (
+              invalidAncestor.parent.parent?.type ===
```

Comment:
this line's ? operator is not covered with test code. but i don't know the case that invalidAncestor is ReturnStatement and invalidAncestor.parent.parent is undefind